lib/metadata_json_deps/runner.rb

```diff
@@ -46,6 +49,7 @@ def run
 
       puts message
       post_to_slack(message) if @use_slack
+      post_to_logs(message)
```

Comment:
You'll need to add an `if @logs_file` like is done on the previous line, to only run the function when a value for the log file has been passed in, as it throws an error.